symphony/content/content.systemauthors.php

```diff
@@ -566,6 +566,18 @@ public function __actionNew()
         if (@array_key_exists('save', $_POST['action']) || @array_key_exists('done', $_POST['action'])) {
             $fields = $_POST['fields'];
 
+            $canCreate = Symphony::Author()->isDeveloper() || Symphony::Author()->isManager();
+
+            if (!$canCreate) {
+                $this->pageAlert(__('You don\'t have the rights to create a new author.'), Alert::ERROR);
+                return;
+            }
+
+            if (Symphony::Author()->isManager() && $fields['user_type'] !== 'author') {
+                $this->pageAlert(__('The user type is invalid. You can only create Authors.'), Alert::ERROR);
+                return;
+            }
+
```

Comment:
I would rather use the `$this->_errors['user_type']` mechanism instead of page alerts.